src/FileDFG/services/FileDFG.jl

```diff
@@ -11,18 +11,25 @@ dfg = initfg()
 # ... Add stuff to graph using either IIF or DFG:
 v1 = addVariable!(dfg, :a, ContinuousScalar, labels = [:POSE], solvable=0)
 # Now save it:
-saveDFG(dfg, "/tmp/saveDFG")
+saveDFG(dfg, "/tmp/saveDFG.tar.gz")
```

Comment:
This is to just subtly push users to use the extension so we can smoothly remove the folder code in the future.

Reply:
agree